vscode/oso/package.json

```diff
@@ -1,16 +1,22 @@
 {
   "name": "oso",
+  "author": "Oso Security Inc.",
   "publisher": "osohq",
-  "displayName": "oso",
-  "description": "Syntax highlighting for .polar files as used by oso.",
+  "license": "Apache-2.0",
+  "displayName": "Oso",
+  "description": "Support for Oso policies written in Polar.",
   "icon": "oso_logo.png",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
maybe 0.2?

Reply:
Eh, this shouldn't be breaking. It should only be adding new highlights.

We'll save 0.2 for when it has LSP : )